src/gpl/test/gpl_aux.py

```diff
@@ -160,7 +160,7 @@ def global_placement(
         if incremental:
             gpl.doIncrementalPlace(1)
         else:
-            gpl.doInitialPlace()
+            gpl.doInitialPlace(1)
```

Comment:
Why 1 and not the default like tcl?

Reply:
because it's `1` in `doIncrementalPlace` and `doNesterovPlace` (above and below). I think it can be a separate PR to change this, but it only effects the testing which already defaults to 1 anyways.

Reply:
@maliberty I wasn't planning on making that PR